src/hooks/users/useWorldUsers.ts

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from "react";
+
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+import { worldUsersSelector } from "utils/selectors";
+
+import { isLoaded } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+
+import { useConnectWorldUsers } from "./useConnectWorldUsers";
+
+export const useWorldUsers = (): {
```

Comment:
[note] This hook was split out of `src/hooks/users/index.ts` into it's own file here.